src/test/spec/auth.rs

```diff
@@ -50,13 +50,7 @@ async fn run_auth_test(test_file: TestFile) {
     for mut test_case in test_file.tests {
         test_case.description = test_case.description.replace('$', "%");
 
-        let skipped_mechanisms = [
-            "GSSAPI",
-            "MONGODB-X509",
-            "PLAIN",
-            "MONGODB-CR",
-            "MONGODB-AWS",
```

Comment:
I forgot to check CI before, but it looks like this test is always failing on async-std since AWS is only enabled on tokio

Reply:
Ah, right. Fixed.